serverless/src/util/database/getDbConnection.js

```diff
@@ -13,9 +13,17 @@ export const getDbConnection = async () => {
   if (dbConnection == null) {
     const dbConnectionConfig = await getDbConnectionConfig()
 
+    const pool = {}
+
+    if (process.env.IS_OFFLINE) {
+      // When running locally set the pool min to 0 to avoid idle connections
+      pool.min = 0
+    }
```

Comment:
Was this the solution to the `sorry, too many clients already` error we'd run into locally sometimes?

Reply:
Yep. it has been driving me crazy recently, and I've been copying this change to all my branches and I haven't seen it in a while